src/classes/database.ts

```diff
@@ -98,8 +103,34 @@ export class Database {
     await this.checkVersionAndMigrations(settings.version);
   }
 
+  private async initMatchmakingQueue() {
+    const collection = this.collection<MatchmakingQueue>(Collections.MATCHMAKING_QUEUE);
+    // Initialize the matchmaking queue if necessary
+    const queue = await collection.findOneAsync({});
+    if (!queue) {
+      await collection.insertAsync({ players: [] }); // Empty initial queue
```

Comment:
Initial testing for me, we can do more here like:

```
  queueId: string; // Unique ID for the queue (game mode, region, etc.)
  gameMode: string; // The game mode for the queue, e.g., 'ranked', 'casual'
  region: string; // Region, e.g., 'NA', 'EU'
  combineRegions: regions; // Combine NA and EU for more players
  players: string[]; // Players waiting in the queue
```

This part can be deleted, because I was testing...

Reply:
Ok, knowing it's a test, it's good to merge. About your suggestion, we're not gonna use regions yet, also, a single region may have several queues (for instance, several monsters queue). I was hoping to have Matchmaking Sessions in the Array instead of players themselves

Reply:
Thinking more about this, I just don't think I want the matchmaking data on the database. It doesn't make sense because if a server restarts we're losing the websocket connections anyway, so all matchmaking info is irrelevant.

This is not blocking for merging the pull request because it can be deleted anyway